### ISSUE TITLE
Question about dataset splitting

### ISSUE THREAD

**Issue by Blosslzy** [TencentYoutuResearch/OpenWorld-DeepFakeAttribution#3]
Thank you for sharing your excellent work! I have a few questions regarding the dataset splitting.
1) As mentioned in the preprocessing section, I noticed that there are two types of '.csv' files: '*.csv' and 'large.csv'. Does this signify the training set and validation set, and does the test set refer to the combination of test sets from various datasets?
2)How can I obtain the following meta-files?
meta_files = [
    'ffpp_c23_meta.csv',
    # 'ffpp_c40_meta.csv',
    # 'ffpp_meta.csv',
    'ffpp_c23_real_meta.csv',
    # 'ffpp_c40_real_meta.csv',
    # 'ffpp_meta_real.csv',
    'celebdf_real_meta.csv',
    'forgerynet_meta.csv',
    'dffd_meta.csv',
    'forgerynir_meta.csv',
]
Thank you very much!!!!
